install/kubernetes/cilium/values.yaml.tmpl

```
@@ -1445,6 +1445,18 @@ hubble:
             #     - destination_pod: ["frontend/nginx-975996d4c-7hhgt"]
             #     excludeFilters: []
             #     end: "2023-10-09T23:59:59-07:00"
+
+  # -- Emit v1.Events related to pods on detection of packet drops.
+  dropEventEmitter:
+    enabled: false
+    # --- Minimum time between emitting same events.
+    interval: 120s
+    # --- Drop reasons to emit events for.
+    # ref: https://docs.cilium.io/en/stable/_api/v1/flow/README/#dropreason
+    reasons:
+    - auth_required
+    - policy_denied
```

nit: indent the list to minimize diff between this template and the generated value file
```suggestion
      - auth_required
      - policy_denied
```

---

Fixed.